assets/js/string-calculator.js

```diff
@@ -0,0 +1,56 @@
+const elements = [...document.getElementsByTagName('span')];
+const windowCalc = document.getElementById('window');
+
+function calculationStart(exception) {
+    let operationCash = exception.split('');
+    let exceptionElementFirst='';
+    let exceptionElementSecond='';
+    let operation='';
+    exceptionElementFirst+=operationCash[0];
+    let operationPointer=true;
+    for(let i=1;i<operationCash.length;i++){
+        if(/\d+|[.]/g.test(operationCash[i]) && operationPointer){
```

Comment:
_Code style_: simplify conditional logic or move code parts to functions

Reply:
fixed